setup.py

```diff
@@ -0,0 +1,20 @@
+"""minimum config to enable distribution via pip and pipenv."""
+
+import setuptools
+
+with open('README.rst', 'r') as fh:
+    LONG_DESCRIPTION = fh.read()
+
+setuptools.setup(
+    name='pelican-plugins',
+    version='0.0.1',
```

Comment:
This should match the latest tag on this repo

Reply:
ok super, I'll add something to keep that in sync. :+1: